app/views/container_service/_main.html.haml

```diff
@@ -3,6 +3,8 @@
   .col-md-12.col-lg-6
     = render :partial => "shared/summary/textual", :locals => {:title => _("Properties"), :items => textual_group_properties}
     = render :partial => "shared/summary/textual_multilabel", :locals => {:title => _("Port Configurations"), :items => textual_group_port_configs}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Labels"), :items => textual_container_labels}
+    = render :partial => "shared/summary/textual", :locals => {:title => _("Selector"), :items => textual_container_selectors}
```

Comment:
Right, among other things, let's also be consistent with kubernetes that names them `labels` and `selector` (for the same reason @abonas mentioned)